views/ViewChart.react.js

```diff
@@ -12,10 +12,13 @@ import GraphTable from './components/GraphTable'
 const ViewChart = ({ graph, classes }) => {
   const { title, description } = graph
   return (
-    <AppLayout title={title?.toUpperCase()}>
-      <Typography variant="title" gutterBottom>
-        {description?.toUpperCase()}
-      </Typography>
+    <AppLayout title={title}>
+      {description && (
+        <Typography variant="body1" gutterBottom>
+          <pre style={{ fontFamily: 'inherit' }}>{description}</pre>
```

Comment:
I think the style object can be applied to the Typography component and it should take precedence over it's built in styling.